error_handler.go

```diff
@@ -3,17 +3,11 @@
 
 package otel // import "go.opentelemetry.io/otel"
 
-// ErrorHandler handles irremediable events.
-type ErrorHandler interface {
-	// DO NOT CHANGE: any modification will not be backwards compatible and
-	// must never be done outside of a new major release.
+import "go.opentelemetry.io/otel/internal/global"
 
-	// Handle handles any error deemed irremediable by an OpenTelemetry
-	// component.
-	Handle(error)
-	// DO NOT CHANGE: any modification will not be backwards compatible and
-	// must never be done outside of a new major release.
-}
+// ErrorHandler handles irremediable events.
+// It's an alias for [global.ErrorHandler].
+type ErrorHandler = global.ErrorHandler
```

Comment:
The documentation for this type is far less user-friendly.

Reply:
https://github.com/open-telemetry/opentelemetry-go/pull/7241#issuecomment-3220639041

For this reason, I will close it.